packages/trie/scripts/view.ts

```diff
@@ -9,21 +9,21 @@ import { _walkTrie } from './asyncWalk.js'
 import type { TrieNode } from '../types.js'
 import type { Debugger } from 'debug'
 
-const debug = _debug('trieview')
-const delimeters = {
+const debug = _debug('trieview') // cspell:disable-line
+const delimiters = {
   0: debug.extend(''),
   1: debug.extend('::'),
   2: debug.extend('::::'),
   3: debug.extend('::::::'),
   4: debug.extend('::::::::'),
   5: debug.extend('::::::::::'),
 } as const
-type Delimeter = keyof typeof delimeters
-const delimiter = (level: Delimeter): void => {
-  delimeters[level]('-'.repeat(50 - level * 2))
+type Delimiter = keyof typeof delimiters
+const delimiter = (level: Delimiter): void => {
+  delimiters[level]('-'.repeat(50 - level * 2))
 }
 const tNode = ['br', 'lf', 'ex', 'rt', 'nl', 'pf', 'vl'] as const
-type TNode = typeof tNode[number]
+type TNode = (typeof tNode)[number]
```

Comment:
Is this update intentional? Not sure what the resulting type is, but seems it could be different

Reply:
Also not sure if the formatting changes later in the file are intentional

Reply:
Not sure, I do not recall changing this. Cspell should not change anything, so this could be caused by the linter (for some reason?)

Reply:
Have reverted